cs50/2017/fall/recover/check50/generate_jpg_data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# get the sizes of the jpegs in bytes
+wc -c *.jpg > sizes.txt
```

Comment:
This seems to be more portable than the equivalent `stat` command.

Reply:
k, but tweak to `wc -c *.jpg | head -n -1 > sizes.txt` to remove last "total" line.